<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBook-Pro-de-Antonio.local" failures="0" tests="1" name="translatorTest.TranslatorParallelTest" time="1.401" errors="1" timestamp="2021-09-16T01:05:28 PET" skipped="0">
  <testcase classname="translatorTest.TranslatorParallelTest" name="TestTranslation" time="1.401">
    <error message="Method translatorTest.TranslatorParallelTest.TestTranslation() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method translatorTest.TranslatorParallelTest.TestTranslation() didn't finish within the time-out 1000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:322)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:280)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:605)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:66)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </error>
  </testcase> <!-- TestTranslation -->
  <system-out/>
</testsuite> <!-- translatorTest.TranslatorParallelTest -->
